### DISCUSSION TITLE
Save Image - Visual Graph

### DISCUSSION THREAD

**Discussion by b1rdi3** [srixivas/PcapXray#73] [Ideas]
First off, solid tool. 

Second, it appears theres only the ability to save the interactive graph, but for some reports it can come out a little messy, whereas the other graph (visualize) gives a cleaner diagram. 
Am I missing how to save just the visualization? or is there a quick fix that can be piped in to support that? 

Thanks

**Comment by srixivas** [comment]
Thanks! All the graphs generated should be available/saved at `<chosen_output_dir>/Report` path selected. Visualize category as image files.
